README.md

```diff
@@ -107,3 +107,25 @@ After installing Rust, enter the `rust` directory and run
 cargo run
 ```
 This will output the available subcommands (which can be viewed as entry points); you can run any of them with e.g. `cargo run mqtt-pub-meta`.
+### Web UI service
+
+The Web UI service provides an interface to setup the dataloggers.
+
+This service is a `Flask` application that uses Jinja2 as template engine and Python 3.10.
+
+<i>Create a new page within `web-ui` service</i>
```

Comment:
Would prefer to use markdown instead of HTML (`*` around text for italics, or just `####` for a heading, which probably also works here)